src/client/sandbox/shadow-ui.js

```diff
@@ -600,6 +619,10 @@ export default class ShadowUI extends SandboxBase {
             ShadowUI._markElementAsShadow(childElement);
     }
 
+    static markCollectionAsShadow (collection) {
```

Comment:
`markAsShadowContainerCollection (collection)`

Reply:
I think that the `Container` word is extra.

Reply:
We have:
`container-collection-flag`,
`isShadowContainerCollection` check,

but 

`getShadowUICollectionLength` function